src/env.js

```diff
@@ -6,6 +6,8 @@ const host =
 const url =
   process.argv.includes("--dev") == 1
     ? "http://localhost:61613/"
+    // for testing on other devices (like androidTV or so...), for example you can use command "npx cloudflared tunnel --url http://localhost:61613" and put the generated url here down
+    // Then start the addon without --dev: "node src/server.js"
```

Comment:
Wouldn't it be better to replace the above URL? Then you can still use `npm run dev` and have the advantage of auto-reloading.